objects/Editors/contained/9207a4/script.lua

```diff
@@ -380,10 +385,27 @@ end
 ---
 cardLoadingScript = "function onLoad(saved_data)\n    if saved_data ~= \"\" then\n        local loaded_data = JSON.decode(saved_data)\n        self.setTable(\"thresholds\", loaded_data.thresholds)\n    end\nend\n-- card loading end"
 
+function ensureScriptState(keys)
+    local state = {}
+    if currentCard.script_state and currentCard.script_state ~= "" then
+        state = JSON.decode(currentCard.script_state)
+    end
+
+    for _, key in ipairs(keys) do
+        local k = tostring(key)
```

Comment:
i think keys is always just a table of strings? or do you envision a world where that's not the case?

Reply:
also it seems that the table is only ever 1 element, so perhaps rewriting this function to just take in a single string would make things a little cleaner?

Reply:
True, it would just be strings, so no need to convert it. And I was just future proofing with making it a loop for multiple keys